Cargo.toml

```diff
@@ -3,6 +3,9 @@ name = "small-powers-of-tau"
 version = "0.1.0"
 edition = "2021"
 authors = ["Kevaundray Wedderburn"]
+description = "Small Powers of Tau"
+repository = "https://github.com/crate-crypto/small-powers-of-tau"
+license = "ISC"
```

Comment:
Can you remove the ISC license and put it as MIT/Apache ?

I haven't read up on the ISC license, but since its not the license referenced in the readme (MIT/Apache), its better to use a separate PR/Issue to initiate changing the license

Reply:
I just deleted the license attribute and file. The compile is not throwing any errors. We are good! :)